src/style.css

```
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 800;
+}
+
+body {
+  padding: 0;
+}
+
+.form {
+  margin-top: 20px;
+  display: flex;
+}
+
+.form__big {
+  height: 70px;
+}
+
+.input-bar {
+  font-weight: bold;
+  font-family: Avenir, sans-serif;
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
```

```suggestion
  background-position: center;
  background-size: cover;
```
Let`s try to make like that